<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WorkplaceForm</class>
 <widget class="QWidget" name="WorkplaceForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QFrame" name="first_frame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>321</width>
       <height>578</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::WinPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0" colspan="2">
       <widget class="QTableView" name="total_tableView">
        <property name="minimumSize">
         <size>
          <width>295</width>
          <height>432</height>
         </size>
        </property>
        <property name="cursor" stdset="0">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">QTableView {
    background-color: #fdfdfd;
    alternate-background-color: #f0f4f8;
    gridline-color: #d1dbe5;
    font: 10pt &quot;Segoe UI&quot;;
    color: #102a43;
}

QHeaderView::section {
    background-color: #1d4ed8;
    color: white;
    padding: 4px;
    border: 1px solid #0f172a;
    font-weight: bold;
}

QTableView::item:selected {
    background-color: #93c5fd;
    color: #0c1a2c;
}

QTableView::item:hover {
    background-color: #dbeafe;
}
</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::WinPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="gridStyle">
         <enum>Qt::DashDotLine</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="sumX_label">
        <property name="minimumSize">
         <size>
          <width>144</width>
          <height>23</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>23</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Times New Roman</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="frameShape">
         <enum>QFrame::WinPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
        <property name="text">
         <string>Ex=</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="sumXY_label">
        <property name="minimumSize">
         <size>
          <width>144</width>
          <height>23</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>23</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Times New Roman</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="frameShape">
         <enum>QFrame::WinPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
        <property name="text">
         <string>E[xy]=</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="sumY_label">
        <property name="minimumSize">
         <size>
          <width>144</width>
          <height>23</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>23</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Times New Roman</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="frameShape">
         <enum>QFrame::WinPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
        <property name="text">
         <string>Ey=</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="meanX_label">
        <property name="minimumSize">
         <size>
          <width>144</width>
          <height>23</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>23</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Times New Roman</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="frameShape">
         <enum>QFrame::WinPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
        <property name="text">
         <string>mean(x)=</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="sumX2_label">
        <property name="minimumSize">
         <size>
          <width>144</width>
          <height>23</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>23</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Times New Roman</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="frameShape">
         <enum>QFrame::WinPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
        <property name="text">
         <string>E[x^2]=</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="meanY_label">
        <property name="minimumSize">
         <size>
          <width>144</width>
          <height>23</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>23</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Times New Roman</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="frameShape">
         <enum>QFrame::WinPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
        <property name="text">
         <string>mean(y)=</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="sumY2_label">
        <property name="minimumSize">
         <size>
          <width>144</width>
          <height>23</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>23</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Times New Roman</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="frameShape">
         <enum>QFrame::WinPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
        <property name="text">
         <string>E[y^2]=</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QFrame" name="second_frame">
     <property name="minimumSize">
      <size>
       <width>322</width>
       <height>578</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::WinPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <widget class="QTableView" name="calculate_tableView">
      <property name="geometry">
       <rect>
        <x>12</x>
        <y>12</y>
        <width>256</width>
        <height>192</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QTableView {
    background-color: #fdfdfd;
    alternate-background-color: #f0f4f8;
    gridline-color: #d1dbe5;
    font: 12pt &quot;Segoe UI&quot;;
    color: #102a43;
}

QHeaderView::section {
    background-color: #1d4ed8;
    color: white;
    padding: 4px;
    border: 1px solid #0f172a;
    font-weight: bold;
}

QTableView::item:selected {
    background-color: #93c5fd;
    color: #0c1a2c;
}

QTableView::item:hover {
    background-color: #dbeafe;
}
</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::WinPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="gridStyle">
       <enum>Qt::DashDotLine</enum>
      </property>
     </widget>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QFrame" name="third_frame">
     <property name="minimumSize">
      <size>
       <width>321</width>
       <height>578</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::WinPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <widget class="QPushButton" name="pushButton_back">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>520</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>131</width>
        <height>41</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>131</width>
        <height>41</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background-color: #DCE1E6;
    color: #102A43;
    border-radius: 8px;
    border: 1px solid #A0A0A0;
    padding: 5px 12px;
}

QPushButton:hover {
    background-color: #C9D4DF;
    color: #0c1a2c;
}

QPushButton:pressed {
    background-color: #AFC3D0;
    border: 1px solid #7A8A99;
    color: #ffffff;
}

QPushButton:checked {
    background-color: #B0E0FF;
    border: 1px solid #3399FF;
    color: #003366;
}
</string>
      </property>
      <property name="text">
       <string>Вернуться</string>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
